Code/src/main/java/nl/utwente/group10/ui/components/lines/Connection.java

```diff
@@ -165,6 +230,17 @@ public final void changed(ObservableValue<? extends Number> observable,
         updateStartEndPositions();
     }
 
+    /** @return the scene-relative Point location of this anchor. */
+    private Point2D getScenePoint(ConnectionAnchor anchor) {
+        double x = anchor.getCenterX();
+        double y = anchor.getCenterY();
+        return anchor.localToScene(x, y);
+    }
+
+    public final boolean isConnected() {
+        return startAnchor.isPresent() && endAnchor.isPresent();
+    }
```

Comment:
Sinds isFullConnected() gerefactored is naar isConnected is deze verwarring ook opgelost, Done